sentry_sdk/integrations/modules.py

```diff
@@ -18,15 +18,29 @@
 _installed_modules = None
 
 
+def _normalize_module_name(name):
```

Comment:
```suggestion
def _normalize_module_name(name: str):
```

Reply:
we can not have types this way, because we still support Python 2.7.